base.gradle

```diff
@@ -209,78 +209,77 @@ dependencies {
     implementation androidLibs.composeUiUtil
     implementation androidLibs.wearPlayServices
 
-    implementation libs.kotlinCoroutines
-    implementation libs.kotlinCoroutinesAndroid
-    implementation libs.kotlinCoroutinesPlayServices
-    implementation libs.kotlinCoroutinesRx
-    implementation libs.deviceNames
-    implementation libs.glide
-    implementation libs.glideOkHttp
-    implementation libs.coil
-    implementation libs.coilCompose
-    implementation libs.okHttp
-    implementation libs.okHttpLogging
-    implementation libs.rxJava
-    implementation libs.rxAndroid
-    implementation libs.rxKoltin
-    implementation libs.rxRelay
-    implementation libs.retrofit
-    implementation libs.retrofitMoshi
-    implementation(libs.retrofitProtobuf) {
+    implementation androidLibs.kotlinCoroutines
+    implementation androidLibs.kotlinCoroutinesAndroid
+    implementation androidLibs.kotlinCoroutinesPlayServices
+    implementation androidLibs.kotlinCoroutinesRx
+    implementation androidLibs.deviceNames
+    implementation androidLibs.glide
+    implementation androidLibs.glideOkHttp
+    implementation androidLibs.coil
+    implementation androidLibs.coilCompose
+    implementation androidLibs.okHttp
+    implementation androidLibs.okHttpLogging
+    implementation androidLibs.rxJava
+    implementation androidLibs.rxAndroid
+    implementation androidLibs.rxKoltin
+    implementation androidLibs.rxRelay
+    implementation androidLibs.retrofit
+    implementation androidLibs.retrofitMoshi
+    implementation(androidLibs.retrofitProtobuf) {
         exclude group: 'com.google.protobuf', module: 'protobuf-java'
     }
-    implementation libs.retrofitRxJava
-    implementation libs.moshi
-    implementation libs.moshiAdapters
-    implementation libs.kotlin
```

Comment:
I created a new Android project and it seems like you no longer define Kotlin as a dependency and let the Gradle plugin do this.